docs/source/institutional-faq.rst

```diff
@@ -0,0 +1,362 @@
+when they first investigate Dask.
+
+.. contents:: :local:
+
+For Management
+--------------
+
+Briefly, what problem does Dask solve for us?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Dask is a general purpose parallel programming solution.
+As such it is used in *many* different ways.
+
+However, the most common problem that Dask solves is connecting Python analysts
+to distributed hardware.  The institutions for whom Dask has the greatest
+impact are those who have a large body of Python users who are accustomed to
+libraries like NumPy, Pandas, Jupyter, Scikit-Learn and others, but want to
+scale those workloads onto a cluster.  Often they also have distributed
+computing resources that are going underused.
+
+Dask removes both technological and cultural barriers to connect Python users
+to computing resources in a way that is native to both the users and IT.
+
+"Help me scale my notebook onto the cluster" is a common pain point for
+institutions today, and it is a common entry point for Dask usage.
+
+
+Is Dask Mature?  Why should we trust it?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes.  While Dask itself is relatively new (it began in 2015) it is built by the
+NumPy, Pandas, Jupyter, Scikit-Learn developer community, which is well trusted.
+
+Dask is a relatively thin wrapper on top of these libraries and,
+as a result, the project can be relatively small and simple.
+It doesn't reinvent a whole new system.
+
+Additionally, this tight integration with the broader technology stack
+gives substantial benefits long term.  For example:
+
+-   Because Pandas maintainers also maintain Dask,
+    when Pandas issues a new releases Dask issues a release in lock-step.
+
+-   Because Scikit-Learn maintainers maintain and use Dask when they train on large clusters,
+    you can be assured that Dask-ML focuses on pragmatic and important
+    solutions like XGBoost integration, and hyper-parameter selection over
+    common model types, and that integration between the two feels natural for
+    novice users.
+
+-   Because Jupyter maintainers also maintain Dask,
+    powerful Jupyter technologies like JupyterHub and JupyterLab are designed
+    with Dask's needs in mind, and new features are pushed quickly to provide a
+    first class modern user experience.
+
+Additionally, Dask is maintained both by a broad community of maintainers,
+as well as substantial institutional support (several full-time employees each)
+by both Anaconda, the company behind the leading data science distribution, and
+NVIDIA, the leading hardware manufacturer of GPUs.  Despite large corporate
+support, Dask remains a community governed project, an is fiscally sponsored by
+NumFOCUS, the same 501c3 that manages NumPy, Pandas, Jupyter, as well as many
+others.
+
+
+Who else uses Dask?
+~~~~~~~~~~~~~~~~~~~
+
+Dask is used by individual researchers in practically every field today.
+It has X monthly downlaods, and is integrated into a many PyData software
```

Comment:
Typo "downloads".

About 2,400,000 downloads/month on PyPI according to https://pypistats.org/packages/dask.

Maybe say "millions of downloads per month" to keep this from going stale?